release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -1109,6 +1131,11 @@ module openconfig-transport-types {
     description "Ethernet compliance code: 100GBASE_DR";
   }
 
+  identity ETH_100GBASE_CU5M {
+    base ETHERNET_PMD_TYPE;
+    description "Ethernet compliance code: 100GBASE_CU5M";
```

Comment:
```suggestion
    description "Ethernet compliance code: 100GBASE_CU5M";
```

Should this be `100GBASE-CR4`?  This is more widely used based on searches I did.

Reply:
Hi!! Yes... seems 100GBASE-CR4 is the standard term.  Found the reference in IEEE: Ehttps://grouper.ieee.org/groups/802/3/bj/public/may12/diminico_01a_0512.pdf

I had put as first version the sample I got from the vendor replied that :-)  Will change to the standard one and add the reference to IEEE.

Reply:
hI @dplore 100GBASE_CR4 already existed in Openconfig, so I just update the reference. Do you think is is also worth updating the description to say it is a DAC?